### ISSUE TITLE
StepInput Invalid styles path

### ISSUE THREAD

**Issue by fkolar** [SAP/fundamental-ngx#3081]
#### Is this a bug, enhancement, or feature request?
BUG

#### Briefly describe your proposal.
I get this error in console: 

```
item-detail:1 
Refused to apply style from 'http://localhost:4200/home/travis/build/SAP/fundamental-ngx/node_modules/fundamental-styles/dist/step-input.css' 
because its MIME type ('text/html') is not a supported
 stylesheet MIME type, and strict MIME checking is enabled.
```

and when checking `fundamental-ngx-core.metadata.json` or  `fundamental-ngx-core.js` and , we do have this path in there. 
like this: `import url(/home/travis/build/SAP/fundamental-ngx/node_modules/fundamental-styles/dist/step-input.css)`

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
latest branch

**Comment by droshev** [comment]
@fkolar i was not able to reproduce it

**Comment by fkolar** [comment]
I will commit my example I am working on I will paste the link in here.

**Comment by fkolar** [comment]
1. Try to clone this repo: https://github.com/fkolar/microfrontend-st/
2. Go to this submodule: https://github.com/fkolar/microfrontend-st/tree/master/microfrontend-apps
3.` npm i`
4. `ng build content-item-app `
5. `ng build common-req-app`
6. `ng serve`
7. open http://localhost:4200/
8. click on rubic cube to get to the detail page
9. open up console and you should see this error:

<img width="1339" alt="Screen Shot 2020-08-25 at 5 39 07 PM" src="https://user-images.githubusercontent.com/17149942/91195989-e4e9e380-e6f9-11ea-84f3-27a02ffbc78c.png">

my current installed versions:

"_id": "@fundamental-ngx/core@0.21.0",
"_id": "fundamental-styles@0.10.0",
"_id": "@sap-theming/theming-base-content@11.1.21",

**Comment by fkolar** [comment]
Another team just asked me about the same error.

**Comment by dimamarksman** [comment]
@fkolar I assume that it was fixed 7 days ago in [this commit](https://github.com/SAP/fundamental-ngx/commit/1ce2b1f443c1c13db1938e4376bfc9bff2a36eb7#diff-8c06c9b95a74c9d493c34335019c8ad3L2)

It should be working starting from `fundamnetal-ngx/core` version 0.22.0-rc.0

**Comment by dimamarksman** [comment]
@salarenko promised to resolve this patch today

**Comment by salarenko** [comment]
Fixed with latest release [Core v0.21.3](https://www.npmjs.com/package/@fundamental-ngx/core/v/0.21.3)
